### ISSUE TITLE
It's not possible to center/middle align a font-icon

### ISSUE THREAD

**Issue by fanturi** [angular/material#2630]
Hi,

I didn't find a way to align a font-icon. Alone in a button or next to a text.
For example:
````html
<md-toolbar class="md-toolbar-tools" layout-align="space-between center">
  <md-button class="md-icon-button" aria-label="Menu" ng-click="appCtrl.toggleMenu()">
    <md-icon md-font-icon="fa fa-bars"></md-icon>
  </md-button>

  <span hide show-gt-md class="md-title"><em>Title</em></span>
      
   <md-button ui-sref="matchs">
     <span class="md-inline-list-icon-label">Site name</span>
     <md-icon md-font-icon="icon-diff-found"></md-icon>
   </md-button>
</md-toolbar>
````

There is another problem, the icon has not the correct color.
"Title" or "Site name" are on white but not the font icon.
Someone has a solution ?

Thanks.

**Comment by pencilcheck** [comment]
Maybe this would help for now?

Written in stylus:
```
md-icon
  .material-icons
    display block !important
```

**Comment by raftalks** [comment]
you can set the size of the icon with the class .md-48 and it worked for me.
```
<md-button class="md-icon-button" aria-label="Menu" ng-click="appCtrl.toggleMenu()">
    <md-icon md-font-icon="fa fa-bars" class="md-48"></md-icon>
</md-button>
```

**Comment by xenoterracide** [comment]
http://stackoverflow.com/q/31840673/206466 `md-48` doesn't work for me, and centering only works if I add `material-icons` to the class

**Comment by ThomasBurleson** [comment]
@fanturi - Please provide a CodePen or Plunkr that demonstrates this issue. Here are some starter demo templates that you can use/fork:

*  CodePen template: http://codepen.io/team/AngularMaterial/pen/BNYppz
*  Plunkr template: http://plnkr.co/edit/yP0gB9iTacIQb48thud8?p=preview.

**Comment by fanturi** [comment]
@ThomasBurleson - http://codepen.io/anon/pen/QbYPeP
You can see that those who have a standard size icon are too high and those with doubled size are too low.

**Comment by cmacdonnacha** [comment]
Just wondering what version this is fixed in? I'm currently using 1.0.1.

**Comment by EladBezalel** [comment]
not merged yet #6043
